src/chat.rs

```diff
@@ -948,6 +996,9 @@ pub struct ChatInfo {
     /// Whether the chat is archived.
     pub archived: bool,
 
+    /// Wether the chat is pinned.
+    pub pinned: bool,
```

Comment:
This is a tri-state but here exposed as bools?  Isn't that a bit weird?  If backwards compatiblity is a problem maybe a new name for the tri-state is more suitable while leaving the archived field untouched but documented as deprecated.  Though given only desktop is consuming this currently I'd hope this wouldn't be necessary yet.

Reply:
I like to have the API like that as it is easier to consume that way and you don't need 2 extra comparisons on the desktop side.

Reply:
i also think, `archived` should be a boolean here, to be in sync with `dc_archive_chat()` and `dc_chat_get_archived()` which also use a tri-state. the ffi-users has to handle the tri-state anyway.